build/images/webui/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:20.04
+RUN apt-get update && apt-get install -y git python3 wget
+RUN wget -O - https://raw.githubusercontent.com/alberthier/git-webui/master/install/installer.sh | sh
+RUN ln -s /usr/bin/python3 /usr/bin/python
+WORKDIR /home/download/network-policy-repository
```

Comment:
let's use `/data/resource-auditing-repo` here as the work directory

later on (not this PR), I think we should overwrite the command args in the yaml manifest and use the `--repo-root` command-line option for `git webui`